ui_testing/testcases/basic_tests/test006_orders.py

```diff
@@ -3078,3 +3078,31 @@ def test090_if_cancel_archive_order_no_order_suborder_analysis_will_archived(sel
             self.orders_page.filter_by_analysis_number(filter_text=analysis_no[i])
             self.assertEqual(len(self.order_page.result_table()) - 1, 0)
 
+    def test091_add_any_number_of_suborders(self):
+        '''
+        Orders: Table with add: Allow user to add any number of the suborders records not only 5 suborders
+        LIMS-5220
+        '''
+        testPlan = TestPlanAPI().create_completed_testplan_random_data()
+        order = self.orders_api.get_order_with_multiple_sub_orders(no_suborders=4)
```

Comment:
this test case will fail if no order with multiple suborders above 4 not found
@0xIslamTaha , so should I ask omnia to modify it.
or it's accepted ?
If accepted you can merge code

Reply:
@ghada-37 @omnia353 
Create an order with 5 suborders using the API and use it here

Reply:
ok